frontend/src/Pages/ProgramOverviewDashboard.tsx

```diff
@@ -330,104 +326,126 @@ export default function ProgramOverviewDashboard() {
 
             {/* classes table */}
             <div className="card p-4">
-                <table className="table w-full table-fixed  mb-4">
-                    <thead className="bg-background">
-                        <tr>
-                            <th className="w-[150px]">
+                <table
+                    className="table-2 mb-4"
+                    style={{
+                        display: 'table',
+                        tableLayout: 'fixed',
+                        width: '100%'
+                    }}
+                >
+                    {/* colgroup allows us to manually set column widths of table-2 as long as style of table is set to display: table*/}
+                    <colgroup>
+                        <col style={{ width: '2%' }} /> {/* Checkbox */}
+                        <col style={{ width: '25%' }} /> {/* Class Name */}
+                        <col style={{ width: '25%' }} /> {/* Instructor Name */}
+                        <col style={{ width: '12%' }} /> {/* Start Date */}
+                        <col style={{ width: '12%' }} /> {/* End Date */}
+                        <col style={{ width: '12%' }} /> {/* Enrollments */}
+                        <col style={{ width: '12%' }} /> {/* Status */}
```

Comment:
@corypride I'm not sure about this, but do we have to define all of these widths? Can we set just the ones that we want minimum/maximum widths for ("Status" and "Checkbox") and let the other ones shrink/grow with the space that's left? 

It seems problematic if we add/remove a column and have to readjust the column widths each time.